index.rst

```diff
@@ -770,6 +770,166 @@ calibrations from separate directories: calibrations shall be read from the
 calibration root directory, and written to the same root directory as other
 butler datasets.
 
+Recipes
+^^^^^^^
+
+Recipes are sets of parameters used to control the operation of a
+`Formatter`. The recipe is provided to the formatter as an
+``lsst.daf.base.PropertySet``, known as the ``additionalData``. Recipes are
+grouped by the relevant storage type and each recipe has a symbolic name which
+may be used to refer to the appropriate set of parameters. Recipes are defined
+in YAML format in ``obs_base/policy/writeRecipes.yaml``; obs packages may
```

Comment:
Maybe say that `readRecipes` are not expected to be needed but could be added later (hence the name of this file)?